sim/scenarios/drop-rate/README.md

```diff
@@ -21,9 +22,15 @@ This scenario has the following configurable properties:
   (in percentage) in the server to client direction. This is a required
   parameter. For example, `--rate_to_client=10`.
 
+* `--burst_to_client`: The maximum number of packets that will be dropped in a
+  row in the server to client direction. This is a required parameter. For
+  example, `--burst_to_client=3`.
```

Comment:
Should we default to `INT_MAX`? Would probably make getting this into the interop runner easier, since we don't have coordinate releases.

Reply:
Can do.